### ISSUE TITLE
Cant generate JSON from localhost address

### ISSUE THREAD

**Issue by Webrow** [BuilderIO/figma-html#85]
As I am developping locally Id like sometimes like to export designs from there. When I click the plugin, no "Download" is started, or JSON is created. (it works on other pages). Any debug I can check or look into?

**Comment by steve8708** [comment]
Hey @Webrow - are you using the figma plugin or the chrome extension to grab your content from localhost?

**Comment by Webrow** [comment]
Yes correct! I installed the chrome extension, go to local served host and try to export it, but no JSON is being generated (no download triggered and no tab)

**Comment by Webrow** [comment]
@steve8708 Not to bump, but is there any debugging you'd like me to do to look into this?

**Comment by steve8708** [comment]
Yeah, two things I'd love to find out

- do you get any errors in the chrome devtools JS console? if so can you share?
- if you something like [ngrok](https://ngrok.com/) to make the host no longer localhost, does importing work?

**Comment by bradleytaftsmith** [comment]
hey @steve8708 I'm also running into the same behavior. 

I get no output on JS console on chrometools (short video [here](https://www.loom.com/share/bc8e2ccdbb3b4721b27f82d11f476598))
I was unable to get the file shown there served simply using ngrok and didn't have time to troubleshoot it. getting the HTML I'm wanting into Figma hosted somewhere is more trouble than this problem is worth.

**Comment by bradleytaftsmith** [comment]
I was able to work around the problem above by using npm [http-server](https://www.npmjs.com/package/http-server) to serve my HTML page I wanted captured locally. I used homebrew to install npm, npm to install http-server, a http-server to serve the file to my browser. Then figma-html was able to handle it and export json as expected.
